app/src/main/java/edu/nextstep/camp/calculator/MainPresenter.kt

```diff
@@ -1,16 +1,24 @@
 package edu.nextstep.camp.calculator
 
-class MainPresenter(private val view: MainContract.View) : MainContract.Presenter {
-    private val calculator = Calculator()
-    private var expression = Expression.EMPTY
+import edu.nextstep.camp.calculator.view.MemoryUIModel
+
+class MainPresenter(
+    private val view: MainContract.View,
+    private val calculator: Calculator = Calculator(),
+    private var expression: Expression = Expression.EMPTY
+) : MainContract.Presenter {
+    private var isMemoryViewDisplayed = true
+    private val _logs = mutableListOf<MemoryUIModel>()
+    private val logs: List<MemoryUIModel>
+        get() = _logs
```

Comment:
LiveData가 아니고, 외부에 immutable한 값을 공개하는 것도 아니기 떄문에 언더바를 사용하지 않아도 됩니다.
아래 정도만 되어도 로직에 큰 문제가 없다는 생각이 드네요 :)

```suggestion
    private val logs = mutableListOf<MemoryUIModel>()
```